extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -232,12 +233,27 @@ export class AmpUserNotification extends AMP.BaseElement {
   }
 
   /**
-   * Get async cid service.
+   * Opts the user out of cid issuance and dismisses the notification.
+   * @private
+   */
+  optoutOfCid_() {
+    return this.getCidService_()
+        .then(cid => cid.optOut())
+        .then(this.dismiss.bind(this), reason => {
+          dev().error('amp-user-notification',
+              'Failed to opt out of Cid', reason);
+          // If optout fails, dismiss notification without persisting.
```

Comment:
This seemed to me as the most reasonable thing to do if opting out fails, thoughts?